projects/cds/README.md

```diff
@@ -0,0 +1,57 @@
+# Spartacus CDS configuration
+
+## CDS Module
+
+To enable the cds module in spartacus, add it to the list of imports in your root module, and create a CDSConfig using the Spartacus ConfigModule. The following is an example: 
+```
+@NgModule({
+  imports: [
+    CdsModule,
```

Comment:
Just a note: after the `feature/COL-168-cms-component` is merged (see this change: https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5360/files#diff-ec25be2b01d8ac3f45ccaf9a2f0bb2feL16), this should say `CdsModule.forRoot()`

Reply:
Ok, I pulled it in and refactored